components/dashboard/src/components/UsageBasedBillingConfig.tsx

```diff
@@ -174,15 +173,7 @@ export default function UsageBasedBillingConfig({ attributionId }: Props) {
         if (!attributionId) {
             return;
         }
-        (async () => {
-            const response = await getGitpodService().server.listUsage({
-                attributionId,
-                order: Ordering.ORDERING_DESCENDING,
-                from: billingCycleFrom.toDate().getTime(),
-                to: Date.now(),
-            });
-            setCurrentUsage(response.creditsUsed);
-        })();
+        getGitpodService().server.getUsageBalance(attributionId).then(setCurrentUsage);
```

Comment:
**thought**: Probably not the best line to add this comment, but thanks for not surfacing any negative value on the dashboard for teams and individual users. 🙏